oauth2_http/java/com/google/auth/oauth2/UserAuthorizer.java

```diff
@@ -343,15 +360,22 @@ public void storeCredentials(String userId, UserCredentials credentials) throws
     }
     AccessToken accessToken = credentials.getAccessToken();
     String acessTokenValue = null;
+    String scopes = null;
     Date expiresBy = null;
     if (accessToken != null) {
       acessTokenValue = accessToken.getTokenValue();
       expiresBy = accessToken.getExpirationTime();
+      List<String> grantedScopes = accessToken.getScopes();
+
+      if (grantedScopes != null) {
```

Comment:
List can be serialized but while deserializing it gets complicated. It is not possible to do an instance of check for List<String>. I get this error: "Cannot perform instanceof check against parameterized type List<String>. Use the form List<?> instead since further generic type information will be erased at runtime"

There are ways to deserialize like using Gson and seems to be another library called jackson that can help. But there is no need to complicate I think.

Reply:
Its just had to be List ) Generic does not exist at runtime )